### DISCUSSION TITLE
possibility to add a description to a deployment which is also visible in Deployment Status overview

### DISCUSSION THREAD

**Discussion by gstoevelaar** [forcedotcom/cli#3064] [Ideas]
When I deploy metadata by the SF CLI in the deployment status overview I only see an id as name. It would be nice if you could define a title or description as sf project deploy start parameter which is then shown in the Deployment Status overview.

**Comment by cristiand391** [comment]
Hey, sorry for the late reply.
This isn't supported by the Metadata API: https://developer.salesforce.com/docs/atlas.en-us.api_meta.meta/api_meta/meta_deploy.htm

If you really want to push this idea maybe take a look at IdeaExchange to see if people would be interested in this feature (and the Metadata API might prioritize it):
https://ideas.salesforce.com/s/

**Comment by smithmd** [comment]
Is there an idea in the exchange for this that I can vote for? This would be very helpful to me as well.
